### ISSUE TITLE
TDS/Vector image missing alt text

### ISSUE THREAD

**Issue by TiffanyMoreside** [telus/tds-core#1702]
## Description

- TDS documentation (notification/feedback icon) indicated that the red exclamation in the circle icon required accessible text.  
- When looking at the code, the SiteImprove link (see below), correct usage showed copy between title tags

This issue directly impacts our team's Product Health score, with a high value assigned to it.

## Reproduction Steps

1. Follow this SiteImprove link [https://my2.siteimprove.com/Inspector/1060596/A11Y/Page?pageId=28217757770&impmd=9101747D1489016CA6C32A5293DD065E#/sia-r43/failed/aHJjSVcxNlA=;YmI1M2MzNDI=/node/0/example](url)
2. Expand the left-hand UI for context on the flagged issue, including WCAG guidance and technical documentation
3. You can view HTML to see where the issue shows up in the code
4. You can also view a copy example

For TDS documentation
1. Notification [https://tds.telus.com/components/index.html#/Feedback%20Indicators?id=notification](url)
2. Feedback Icon [https://tds.telus.com/components/index.html#/Icons?id=feedbackicon](url)

## Recommendation

- Is there a way to build the accessible text into the icon so that when it's uploaded by development teams, it's already correctly labelled and universally rolled out? (I'm not dev or design so I'm not sure how these updates get made site wide)
## Meta

- TDS component version:  @tds/core-feedback-icon@1.1.0
- Willing to develop solution: No (wouldn't know how to)
- High impact: No, but impacts product health

## Screenshots

<img width="330" alt="Error notification:SVG icon" src="https://user-images.githubusercontent.com/86737622/151069864-f9b260c6-8102-4e64-b6fc-898d4091f28f.png">
![Description of error per SiteImprove](https://user-images.githubusercontent.com/86737622/151069882-c57475ea-ba84-469e-9587-2552e3692b39.png)

<img width="581" alt="SVG - code missing title tag:accessible text " src="https://user-images.githubusercontent.com/86737622/151069888-b3b2cade-2239-49b7-84e2-d6c6939822a6.png">

![TDS Feedback Icon - accessible text requriement ](https://user-images.githubusercontent.com/86737622/151069906-0ccf2ca9-a66b-4731-878e-3dce561fa460.png)

- Include any relevant screenshots
![Screen Shot 2022-01-24 at 7 27 38 PM](https://user-images.githubusercontent.com/86737622/151069985-20533423-57e3-4a2f-a24b-b9fc4fa416ff.png)

**Comment by billtelus** [comment]
hey @Tiffany (she/her) did you had a chance to add the props copy ?
I think as mentioned in the tds doc you've to do that for the behaviour you want...
https://tds.telus.com/components/index.html#/Icons?id=feedbackicon

here's a code snippet
```
<NotificationSuccess copy={{ a11yText: 'Success, you have added custom accessibile text' }} />
<NotificationWarning copy="fr" />
```

and check this one too found at: https://tds.telus.com/components/index.html#/Feedback%20Indicators?id=notification

```
<Notification variant="success" dismissible onExit={() => alert('I have exited')} copy="en">
  <Text bold>Your password was updated successfully.</Text> Manage your profile at{' '}
  <Link href="http://telus.com">TELUS.com</Link>
</Notification>
```

> I'll close the issue for now, and feel free to `reopen` it if what I said does not fit your need.

**Comment by owestinTelus** [comment]
It looks like the recommendation was to include a default label for the icons that require descriptive text. 
NotificationSuccess could have a default label "success"
NotificationError could have a default label "error"

Developers should be able to override the default label to improve the experience. 

Another option is to enforce the use of the label and trust content folks, designers, and devs to add the correct label during implementation.

**Comment by owestinTelus** [comment]
Please reopen the issue if either of these options are viable.

**Comment by donnavitan** [comment]
For the recommendation to include a default name for icons with required accessible text, let's use the following:
- NotificationSuccess = Success
- NotificationError = Error
- NotificationWarning = Warning
- LocationSuccess = Available location
- LocationIneligible = Unavailable location
- LocationAdd = Add location


For future reference, we should make sure we have icon names available so that the filename isn't always used as it doesn't reflect a readable name.

@owestinTelus what do you think?

**Comment by owestinTelus** [comment]
Thank you for all of those recommendations Donna.

**Comment by billtelus** [comment]
Thanks Donna. It has been implemented.
